forPublication/vsm.md

```diff
@@ -48,39 +42,37 @@ Another client had a Product Owner signoff task which on average took several da
+Still, certain steps do always take place and lead to a picture as below, usually mirroring the CI/CD pipeline with all the human tasks required for each step.
 #### Should I use specialized tooling?
 Tooling exists and can definitely make automation of the calculations easier, as well as repeatability. 
-I find the results are better using the walls though. The visual aspect of it is important 
-and through collaborating interactively important insights are shared and a common understanding is reached.
-
+I find the results are better using the walls though.
+The visual aspect of it is important and through collaborating interactively important insights are shared and a common understanding is reached.
 #### Other contexts where Value Stream Mapping can be provide Value
-Creating a DevOps, CI/CD pipeline. Finding bottlenecks and understanding the impact of (external) development dependencies.
+Creating a DevOps, CI/CD pipeline.
+Finding bottlenecks and understanding the impact of (external) development dependencies.
 All very useful.
-
 #### Downsides and when to not use
-Consider not using the technique if it you expect it could be used to measure the performance of a team for accountability
- reasons or if psychological safety is lacking. Besides being an undesirable situation it will not work. collaboration 
- spirit and creativity will not be there.   <br/><br/> 
-The numbers you calculate are an valuable indication, but are not exact and hard numbers. Also, make sure that creating
-the VSM doesn't become waste by spending enormous amounts of time gathering the data. Estimates will usually do fine!  
-
+Consider not using the technique if it you expect it could be used to measure the performance of a team for accountability reasons or if psychological safety is lacking.
+Besides being an undesirable situation it will not work. 
+collaboration spirit and creativity will not be there.
+<br/><br/> 
+The numbers you calculate are an valuable indication, but are not exact and hard numbers.
+Also, make sure that creating the VSM doesn't become waste by spending enormous amounts of time gathering the data.
+Estimates will usually do fine!  
 #### Conclusions
-VSM's can help you make inefficiencies in the development process visible. In my opinion it is an essential element in 
-the toolbox of any coach. Furthermore, for teams it tends to be inspiring and can help kickstart a culture of continuous
-improvement. Last but not least, it can make team progress and coaching results more tangible. Try it!
-
-![](maxresdefault.jpg)
+VSM's can help you make inefficiencies in the development process visible.
+In my opinion it is an essential element in the toolbox of any coach.
+Furthermore, for teams it tends to be inspiring and can help kickstart a culture of continuous improvement.
+Last but not least, it can make team progress and coaching results more tangible.
+Try it!
```

Comment:
Nice, I like it. I think it's a good approach to tell about your experience using this tool and what it can do for you as a coach and for the teams you're collaborating with on this!
I do think you need a little bit more background on what it and how to use it, for this to make sense for the readers.
I am not sure how to do that though, without making the blog about that. So you have a challenge :-)